modules/apps/commerce/commerce-frontend-impl/src/main/java/com/liferay/commerce/frontend/internal/cart/model/Cart.java

```diff
@@ -23,14 +23,15 @@
 
 	public Cart(
 		String detailsUrl, long orderId, List<Product> products,
-		Summary summary, boolean valid) {
+		Summary summary, boolean valid, OrderStatusInfo orderStatusInfo) {
 
 		_detailsUrl = detailsUrl;
 		_orderId = orderId;
 		_products = products;
 		_summary = summary;
 		_success = true;
 		_valid = valid;
+		_orderStatusInfo = orderStatusInfo;
```

Comment:
Can we sort this by alphabetical order? if so, we should sort the function parameters above also.
```suggestion
		_detailsUrl = detailsUrl;
		_orderId = orderId;
		_orderStatusInfo = orderStatusInfo;
		_products = products;
		_summary = summary;
		_success = true;
		_valid = valid;
```

Reply:
If I change it, I also have to change the constructor attributes order (otherwise the sf throws an error).
It makes totally sense but it means that all the Cart usages need to be updated by passing the orderStatusInfo data.

This involves some BE work. I'll update the jira ticket